java/api/src/main/java/io/ray/api/call/PyActorCreator.java

```diff
@@ -14,6 +14,11 @@ public PyActorCreator(PyActorClass pyActorClass, Object[] args) {
     this.args = args;
   }
 
+  public PyActorCreator setAsync(boolean enabled) {
```

Comment:
```suggestion
  public PyActorCreator setAsync(boolean isAsync) {
```

I think `isAsync` is better than `enabled` since it's not a switch that both on and off are OK.